[2025-02-10T09:26:40.109+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-10T09:26:40.120+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: run_e2e_tests.start_e2e_tests manual__2025-02-10T09:26:37.618016+00:00 [queued]>
[2025-02-10T09:26:40.126+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: run_e2e_tests.start_e2e_tests manual__2025-02-10T09:26:37.618016+00:00 [queued]>
[2025-02-10T09:26:40.126+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-10T09:26:40.138+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): start_e2e_tests> on 2025-02-10 09:26:37.618016+00:00
[2025-02-10T09:26:40.157+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=14504) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-10T09:26:40.159+0000] {standard_task_runner.py:72} INFO - Started process 14505 to run task
[2025-02-10T09:26:40.159+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'run_e2e_tests', 'start_e2e_tests', 'manual__2025-02-10T09:26:37.618016+00:00', '--job-id', '539', '--raw', '--subdir', 'DAGS_FOLDER/generate_dags_from_yaml.py', '--cfg-path', '/tmp/tmp01x7z_f6']
[2025-02-10T09:26:40.160+0000] {standard_task_runner.py:105} INFO - Job 539: Subtask start_e2e_tests
[2025-02-10T09:26:40.198+0000] {task_command.py:467} INFO - Running <TaskInstance: run_e2e_tests.start_e2e_tests manual__2025-02-10T09:26:37.618016+00:00 [running]> on host fbc98e9fe6bb
[2025-02-10T09:26:40.262+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='run_e2e_tests' AIRFLOW_CTX_TASK_ID='start_e2e_tests' AIRFLOW_CTX_EXECUTION_DATE='2025-02-10T09:26:37.618016+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-10T09:26:37.618016+00:00'
[2025-02-10T09:26:40.264+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-10T09:26:40.265+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-10T09:26:40.265+0000] {logging_mixin.py:190} INFO - Current task name:start_e2e_tests state:running start_date:2025-02-10 09:26:40.121355+00:00
[2025-02-10T09:26:40.265+0000] {logging_mixin.py:190} INFO - Dag name:run_e2e_tests and current dag run status:running
[2025-02-10T09:26:40.266+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-10T09:26:40.267+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-10T09:26:40.268+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'Starting end-to-end tests...'; sleep 10; echo 'Tests ran successfully...'"]
[2025-02-10T09:26:40.278+0000] {subprocess.py:99} INFO - Output:
[2025-02-10T09:26:40.281+0000] {subprocess.py:106} INFO - Starting end-to-end tests...
[2025-02-10T09:26:50.285+0000] {subprocess.py:106} INFO - Tests ran successfully...
[2025-02-10T09:26:50.290+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-10T09:26:50.355+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-10T09:26:50.356+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=run_e2e_tests, task_id=start_e2e_tests, run_id=manual__2025-02-10T09:26:37.618016+00:00, execution_date=20250210T092637, start_date=20250210T092640, end_date=20250210T092650
[2025-02-10T09:26:50.373+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-10T09:26:50.374+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-10T09:26:50.374+0000] {logging_mixin.py:190} INFO - Dag name:run_e2e_tests queued_at:2025-02-10 09:26:37.636526+00:00
[2025-02-10T09:26:50.374+0000] {logging_mixin.py:190} INFO - Task hostname:fbc98e9fe6bb operator:BashOperator
[2025-02-10T09:26:50.393+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-10T09:26:50.423+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-10T09:26:50.424+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
